docs/lib-documentation.md

```diff
@@ -4,6 +4,22 @@ This documentation describes how to interact with the Kurtosis API from within a
 
 _Found a bug? File it on [the repo](https://github.com/kurtosis-tech/kurtosis-client/issues)!_
 
+LambdaModuleContext
+-------------------
+<!-- TODO need to add docs for what Kurtosis modules are -->
+This Kurtosis-provided class is the lowest-level representation of a Lambda Kurtosis module - a Kurtosis module that has exactly one function.
+
+### execute(String argsJson) -\> String
+Executes the function packaged inside the Kurtosis Lambda module with the given JSON-serialized args.
+
+**Args**
+
+An arbitrary JSON-serialized object containing args to the Lambda function. Consult the documentation for the module you're using to determine what this should contain.
+
+**Returns**
+
+An arbitrary JSON-serialized object containing the results of executing the Lambda function. Consult the documentation for the module you're using to determine what this will contain.
```

Comment:
Same here in returns.

Reply:
for return types with a single arg, I tend not to name it. That said, I like the clarity so will do!